cfgov/unprocessed/css/organisms/filterable-list-controls.less

```diff
@@ -0,0 +1,40 @@
+/* topdoc
+  name: Filterable List Controls
+  family: cf-organisms
+  patterns:
+  - name: Filterable List Controls
+    markup: |
+      <div class="o-filterable-list-controls">
+          <div class="m-expandable
+                      m-expandable__borders
+                      m-expandable__midtone
+                      m-expandable__expanded">
+              …
+          </div>
+          <div class="m-notification"></div>
+      </div>
+    codenotes:
+      - |
+        Pattern structure
+        -----------------
+        .o-filterable-list-controls
+          .m-expandable
+            [...]
+          .m-notification
+            [...]
+  tags:
+  - cf-organisms
+*/
+
+.o-filterable-list-controls {
+    // Add gap between expandable and notification.
+    .m-notification {
+        margin-top: unit( @grid_gutter-width / @base-font-size-px, em );
```

Comment:
Hm, initially that sounds like a great idea, but giving it further thought I now I see one sticking point that I think is a deal breaker. Let me know what you think. 
The benefits are:
- All variations of notification are in notification itself.
- Allows `filterable-list-controls.less` to be removed.
- notification constructor on the server-side needs an additional parameter to set the modifier. Again this means all variations of the notification are set in the notification itself, but the downside may be that it doesn't make it clear where those variations are used.

The sticking point is this: An organism is a set of molecules and atoms. The molecules/atoms don't know about their margin, as the parent organism would have the responsibility to set that. However, the molecules could set their padding. This is fine *unless* the molecules have a border and background that would need both a margin and a padding. In that case the margin would go in the parent organism and the padding would be in the molecule.